### ISSUE TITLE
Add DEV user for local development under Ubuntu

### ISSUE THREAD

**Issue by ceckoslab** [basicrum/backoffice#28]
I started experiencing problems with file permissions when I run composer or scaffolding scripts.

The newly created files had read only access from host/IDE. I think that the reason is that they are created from root user.

I found a workaround that does the trick and I will try to implement it only for local environment.

What worked for me under Ubunutu:

```
docker exec -it basicrum_bo_php bash
bash-4.4# addgroup dev -g 999
bash-4.4# adduser dev -g dev -h /home/dev -D
```

I will consider if I integrate this directly in basicrum_bo_php container or even better having separate container for local environment from which I can run composer and other commands.

**Comment by ceckoslab** [comment]
I also it is good to add:
```
export COMPOSER_MEMORY_LIMIT=-1
```

because otherwise `composer require ...` fails with:
```
Fatal error: Allowed memory size of 1610612736 bytes exhausted (tried to allocate 20480 bytes) in phar:///usr/local/bin/composer/src/Composer/DependencyResolver/DefaultPolicy.php on line 71
```

**Comment by ceckoslab** [comment]
Closing it because because @eliskovets solved it in #4
